incubator/hnc/internal/reconcilers/anchor.go

```diff
@@ -75,9 +75,9 @@ func (r *AnchorReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctr
 	// Report "Forbidden" state and early exit if the namespace is not allowed to have subnamespaces
 	// but has bypassed the webhook and successfully created the anchor. Forbidden anchors won't have
 	// finalizers.
-	// TODO refactor/split the EX map for 1) reconciler exclusion and 2) subnamespaces exclusion
-	// purposes. See issue: https://github.com/kubernetes-sigs/multi-tenancy/issues/495
-	if config.EX[pnm] {
+	// TODO refactor/split the ExcludedNamespaces map for 1) reconciler exclusion and 2) subnamespaces exclusion
+	//  purposes. See issue: https://github.com/kubernetes-sigs/multi-tenancy/issues/495
```

Comment:
Extra space at the beginning of the line

Reply:
This indention is on purpose for IntelliJ to highlight the TODO paragraph since this line is part of the TODO. Otherwise only the first line of TODO will be highlighted.

Reply:
Ack, thanks